src/docfx/lib/HttpClientUtility.cs

```diff
@@ -11,49 +11,36 @@ namespace Microsoft.Docs.Build
 {
     internal static class HttpClientUtility
     {
-        private const int RetryCount = 3;
-        private const int RetryInterval = 1000;
         private static readonly HttpClient s_httpClient = new HttpClient();
 
-        public static async Task<HttpResponseMessage> GetAsync(string requestUri, Config config, string etag = null)
+        public static async Task<HttpResponseMessage> GetAsync(string requestUri, Config config, EntityTagHeaderValue etag = null)
```

Comment:
why the change from `string` to `EntityTagHeaderValue`?

Reply:
For `PutAsync`, I can avoid `EntityTagHeaderValue`=>`stirng`=>`EntityTagHeaderValue` conversion between Get and Put. To keep consistent, also changed `GetAsync`.